django/test/runner.py

```diff
@@ -8,6 +9,49 @@
 from django.test import SimpleTestCase, TestCase
 from django.test.utils import setup_test_environment, teardown_test_environment
 from django.utils.datastructures import OrderedSet
+from django.utils.six import StringIO
+
+
+class DebugSQLTextTestResult(unittest._TextTestResult):
```

Comment:
`unittest._TextTestResult` is deprecated. You can use `unittest.TextTestResult` directly.